docs/topics/mta-requirements-for-java-analyzer.adoc

```diff
@@ -8,7 +8,9 @@
 
 There is a minimum required for the Java analyzer, and as a result, the  discovery task.
 
-The minimum requirement is 1.5 GiB. However, this minimum requirement is not unadvisable. The suggested requirement is a minimum of 2 GiB.
+The suggested requirement is a default minimum of 2 GiB. 
+
+The minimum requirement can be lowered to  1.5 GiB. However, this minimum requirement is not recommended. 
 
 This minimum requirement can be set in `provider_java_container_limits_memory` and `provider_java_container_requests_memory` options in the Tackle custom resource.
```

Comment:
The minimum requirement is already set to 2GB if you would like to increase, you can change the values, or something like that. 

I would suggest also that we say something like "to gurantee scheduling has the correct space, requests and limits should be the same.

Reply:
Updated